components/developer/gdb/Makefile

```diff
@@ -48,7 +48,7 @@ COMPONENT_PROJECT_URL=	https://www.gnu.org/software/gdb/
 CONFIGURE_DEFAULT_DIRS=no
 
 # Set python version used by this product
-PYTHON_VERSION=3.7
+PYTHON_VERSION=3.9
```

Comment:
Please remove this line. `PYTHON_VERSION` is by default set to `3.9`.

Reply:
Done